test/integration/serverless/log_normalize.py

```diff
@@ -159,11 +193,28 @@ def _foreach(log):
         logs = json.loads(log, strict=False)
         for log_item in logs:
             fn(log_item)
-        return json.dumps(logs)
+        return json.dumps(logs, sort_keys=True)
 
     return _foreach
 
 
+def flat_map(fn):
```

Comment:
Sorry, this is suuuuper nit-picky. Can we change this to `flatmap` instead? When I wrote this, I took great care to make sure all the normalizing functions contain exactly 7 characters.